wgsl/index.bs

```diff
@@ -1667,8 +1667,11 @@ Extension names are not [=identifiers=]: they do not [=resolves|resolve=] to [=d
   <thead>
     <tr><th style="width:30%">WGSL language extension
         <th>Description
-    <tr><td colspan=2 class=note><span class="marker">Note:</span> No [=language extensions=] are currently defined.
   </thead>
+  <tr><td>pointer_composite_access
+      <td>
+      Supports dereferencing a pointer of [=composite=] type using the same
```

Comment:
I think too many people will think "dereference" means perform the memory access.

Suggest

> Support [[#composite-value-decomposition-expr|composite-value decomposition expressions]] where the root expression is a pointer, yielding a reference.

>  For example, if `p` is a pointer to a structure with member `m`, then `p.m` is a reference to the memory locations for `m` inside the structure `p` points to.

> Similarly, if `pa` is a pointer to an array, then `pa[i]` is a reference to the memory locations for the i'th element of the array `pa` points to.